cmd/package-operator-manager/bootstrap/bootstrap_test.go

```diff
@@ -127,6 +124,7 @@ func TestBootstrapper_bootstrap(t *testing.T) {
 	ctx, cancel := context.WithTimeout(
 		context.Background(), testBootstrapTimeout)
 	defer cancel()
+	var runManagerCtx context.Context
```

Comment:
If this is moved down here, then I'd argue that `runManagerCalled` should be too